cats/CI_api_query.py

```diff
@@ -41,12 +37,14 @@ def get_tuple(
     session = requests_cache.CachedSession('cats_cache', use_temp=True)
     # get the carbon intensity api data
 
-    r = session.get(request_url(dt, postcode))
+    API = API_interfaces["carbonintensity.org.uk"] # TODO give choice of API to user
```

Comment:
I think the `APIInterface` should be passed to `get_CI_forecast` rather than set here. In other word this function should not need any information about _which_ interface it's using.

When we make the choice of API configurable (e.g. via a command line flag), we can do this in the main script.

Reply:
Not fussed either way, changed it in a685996156ccb3877de068a2c0d4bd7535fe10b6